### PR TITLE
security/crowdsec 1.0.8

### PR DESCRIPTION
**Pull request by mmetc**
* Enable use_wal, remove warning
* Randomize cron execution over 5 minutes
* Refact javascript
* Fix initial service start with no pending hub updates (1.6.1)

### CODE REVIEW

**Comment by Monviech** [comment]
Hey can you implement this too in your latest update?
https://github.com/opnsense/plugins/pull/3910

**Comment by mmetc** [comment]
Oh sure, thanks. But I have a note in my local todo that says the max size of a tag is 63 characters... where did you get the 30 from?

**Comment by Monviech** [comment]
It was just an arbitrary number to limit the size of the string. If a tag can be 63 len then it's the better choice.

**Comment by mmetc** [comment]
So 63 it is. I'm pretty sure -/_ are allowed (dots are not) but since the official docs don't mention anything, some tooling might be stricter and I'll stay on the safe side.

Thanks

File: security/crowdsec/pkg-descr [hunk 1]
```diff
@@ -8,6 +8,14 @@ WWW: https://crowdsec.net/
 Plugin Changelog
 ================
 
+1.0.8
+
+* Enable use_wal, remove warning
+* Randomize cron execution over 5 minutes
+* Refact javascript
```
Comment (fichtner) [inline security/crowdsec/pkg-descr:15]:
```suggestion
* Refactor javascript
```

**Review by fichtner — APPROVED** [review]
LGTM, just a typo in the changelog

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**